Quaver.Shared/Screens/Edit/Plugins/Timing/EditorTimingPointPanel.cs

```diff
@@ -186,7 +192,31 @@ private void DrawRemoveButton()
                         SelectedTimingPoints.Add(point);
                 }
 
-                NeedsToScroll = true;
+                NeedsToScrollToFirstSelectedPoint = true;
+            }
+        }
+
+        /// <summary>
+        /// </summary>
+        private void DrawGoToCurrentTimingPointButton()
+        {
+            if (ImGui.Button("Go to current timing point"))
+            {
+                var currentPoint = Screen.WorkingMap.GetTimingPointAt(Screen.Track.Time);
+                if (currentPoint != null)
+                {
+                    SelectedTimingPoints.Add(currentPoint);
```

Comment:
This should probably clear the currently selected timing points before adding the new one? Otherwise it'll select multiple. Some goes for both panels.

Reply:
That's intended and documented in the tooltip a few lines below. The reason for that is to make selecting a range easier. One scenario would be going to a point and selecting it, scrolling to the end of the section you want to select in the editor and clicking the "Select SV". You can then unselect and Shift+Click it again to select the entire range. Another scenario is simply to scroll to different locations in the map and selecting each SV by scrolling to it in the editor and clicking the button, removing the need to figure out which one is which just from the numbers in the table alone. I should probably clarify that intention in the tooltip. Another way to clarify the intention would be to add a "n SV points selected" label somewhere.

Reply:
The button itself could just be phrased a bit better if it has this behavior. For instance, "Go To Objects" in the editor means that every other object gets deselected and the new ones are selected.